content/docs/languages/go/basics.md

```diff
@@ -356,21 +359,22 @@ do this for our `RouteGuide` service:
 
 ```go
 flag.Parse()
-lis, err := net.Listen("tcp", fmt.Sprintf(":%d", *port))
+lis, err := net.Listen("tcp", fmt.Sprintf("localhost:%d", *port))
 if err != nil {
-        log.Fatalf("failed to listen: %v", err)
+  log.Fatalf("failed to listen: %v", err)
 }
-grpcServer := grpc.NewServer()
-pb.RegisterRouteGuideServer(grpcServer, &routeGuideServer{})
-... // determine whether to use TLS
+var opts []grpc.ServerOption
+...
+grpcServer := grpc.NewServer(opts...)
```

Comment:
I added the `var opts` and the `opts` as argument to better match the example code in the repo. Ok?

Reply:
Either way.  It adds clutter to a trivial example, but it's more realistic, since nobody should be using insecure (no creds) in practice.